lib/tasks/signup.rake

```diff
@@ -14,6 +14,7 @@ namespace :signup do
       user = User.create_with(password: ENV["LAGO_ORG_USER_PASSWORD"])
         .find_or_create_by!(email: ENV["LAGO_ORG_USER_EMAIL"])
       organization = Organization.find_or_create_by!(name: ENV["LAGO_ORG_NAME"])
+      billing_entity = BillingEntity.find_or_create_by!(organization:, name: ENV["LAGO_ORG_NAME"], code: ENV["LAGO_ORG_NAME"].parameterize)
```

Comment:
For consistency lets ensure this first billing entity has the same id as the organization.

```suggestion
      billing_entity = BillingEntity.find_or_create_by!(id: organization.id, organization:, name: ENV["LAGO_ORG_NAME"], code: ENV["LAGO_ORG_NAME"].parameterize)
```